test/tests/p5.Helpers.js

```diff
@@ -0,0 +1,35 @@
+const expect = chai.expect;
+
+describe('Testing helpers function', function () {
+  it('p5.prototype.freqToMidi helper function', function () {
+    const midi = p5.prototype.freqToMidi(880);
+    expect(midi).to.equal(81);
+  });
+
+  it('p5.prototype.midiToFreq helper function', function () {
+    const freq = p5.prototype.midiToFreq(720);
```

Comment:
720 is outside of the typical MIDI (and frequency) range, which goes from 0 to 127, might be best to use a number within that range?

Reply:
ok

Reply:
Done